### ISSUE TITLE
I/O Ref description of Pump flow rate schedule should explain that it sets max flow not actual flow

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4274]
User complains that his pump flow rate schedule to vary flow on a variable speed condenser pump is not working.  Comments in ticket say that condenser pump needs to be on supply side of Condenser Loop but user has them on the demand side "after" the chiller condenser.  Ticket says that there should be an error message issued but when you run the file with x8.1 the file runs successfully and no such error is issued.
Helpdesk Ticket 8828
User file
AM3_Testing2.idf

**Comment by Myoldmopar** [comment]
@mjwitte Trying to evaluate this issue.  The file is referencing a `Schedule:File` with path:

```
N:\FS399-EnergyPlus\Defects\Helpdesk Tickets Waiting for CR\Ticket 8828\plantload.txt
```

and the file isn't in the DefectFiles folder on [DevSupport](https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/4000s/4274).

Would you happen to have this available on your N drive and email to me?

**Comment by mjwitte** [comment]
@Myoldmopar It's there now.

**Comment by Myoldmopar** [comment]
Hmm, the flow schedule appears to be enforced properly in the code when the loop solver is querying the pump for available pump capacity, but then it looks like it is overridden by the needs of other components.  Maybe.  Still looking.

**Comment by Myoldmopar** [comment]
OK, not sure again.  

I was playing around with the PlantLoadProfile example file and added a flow schedule to the pump to reduce the flow for part of the day.  Initially, I didn't see any difference.  But then I realized the pump was already running below the (rated flow rate \* fraction) modified value.  So then I changed the fraction even lower, and it started showing the difference, and everything was working well.

Looking at the rated sizes, it looks like the pump here may have been seeing the same behaivor.  So then I cranked down the schedule fraction on the example file, and was able to get it to reduce when the chiller runs on the condenser loop.  There are some other issues going on with this example file, but I believe this verifies the schedule _does_ work, even when the pump is on the demand side.

I'm proposing closing this without any modification.  @mjwitte, any thoughts before I do?

**Comment by mjwitte** [comment]
@Myoldmopar The rest of the story here is that the pump placement in the user's file violates some of the rules in the Engineering Reference.  Are these no longer valid?

Summary of Pump Rules
Pumps in Plant Loop can be on the supply side or demand side
A Pump, if present, in the demand side of plant loop must be the first component of the inlet branch.
Pumps in Condenser loop must be on supply side

And the I/O Ref description of the pump schedule needs more explanation to make it clear that this is only changing the max flow rate and the pump could run below that.

Field: Pump Flow Rate Schedule Name
The schedule values modify the value for Rated Flow Rate of the pump on a time basis. The default is that the pump is ON and runs according to its other
operational requirements specified above. This schedule is for special, not typical pump operations.

**Comment by mjwitte** [comment]
@Myoldmopar read the previous comment in github, I messed it up the first time.

**Comment by Myoldmopar** [comment]
@mjwitte The pump placement rules are _probably_ relaxed enough that it should work fine.  Pumps are treated much better after the pump re-write.  That said, the more relaxed placement rules have not been exercised enough where I want to open up the can of worms of approving it to be used by everyone.  I'll clean up the text for the pump flow rate schedule name to close this.

**Comment by Myoldmopar** [comment]
Fixed by #5198
